### DISCUSSION TITLE
How to remove environment?

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [community/community#45588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I remove environments? In the repository settings the option does not appear.

![Screenshot_20230127-204438](https://user-images.githubusercontent.com/101356556/215226807-803c9685-2dba-4704-846a-a58ea78e087a.jpg)

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @sebastianjnuwu 

In `settings` it has an `environment` section, just go there and delete. See the example:

https://scribehow.com/shared/Github_Workflow_Delete_environment__0Xik8DdSQLqcaG8jnrz-TQ

**Reply by sebastianjnuwu** [reply]
This configuration does not appear for me
![Screenshot_20230127-221414](https://user-images.githubusercontent.com/101356556/215233637-53444aad-4d93-4412-b04f-7c7070f20e09.jpg)

**Reply by SilasRodrigues19** [reply]
Try access by URL

`/settings/environments`

**Reply by sebastianjnuwu** [reply]
page 404

**Reply by sebastianjnuwu** [reply]
![Screenshot_20230128-071829](https://user-images.githubusercontent.com/101356556/215261344-939de8ea-a42a-495a-b4de-edb0c062b894.jpg)
